src/fileimpl.h

```diff
@@ -116,6 +116,50 @@ namespace zim
       bool is_multiPart() const;
   };
 
+
+  template<typename IMPL>
+  std::pair<bool, article_index_t> findx(IMPL& impl, char ns, const std::string& url)
```

Comment:
The name `findx` was fine in the scope of `FileImpl`, but it doesn't look right in the scope of the entire `zim` namespace (even though it is not exposed in the public API).